src/circuit/gadget/ecc/chip/mul.rs

```diff
@@ -131,7 +132,6 @@ impl Config {
         self.hi_config.create_gate(meta);
         self.lo_config.create_gate(meta);
         self.complete_config.create_gate(meta);
-        self.overflow_config.create_gate(meta);
```

Comment:
Refactoring `mul::overflow::Config` first from `mul::Config` causes gate reorderings. I'll see if there's a way to prevent this...